apps/auction/routes.coffee

```diff
@@ -29,10 +29,9 @@ setupUser = (user, auction) ->
           user.set 'registered_to_bid', boolean
         error: ->
           user.set 'registered_to_bid', false
+      ).then( ([ bidder ]) ->
+        user.set 'qualified_for_bidding', bidder?.qualified_for_bidding
```

Comment:
Yeah, you're right! CoffeeScript, innit?

Okay, thanks for looking into this. With that context, looks 👍